test/solveEquation/solveEquation.test.js

```diff
@@ -46,9 +45,9 @@ describe('solveEquation for =', function () {
     ['9x + 4 - 3 = 2x', 'x = -1/7'],
     ['9x + 4 - 3 = -2x', 'x = -1/11'],
     ['5x + (1/2)x = 27 ', 'x = 54/11'],
-    ['2x/3 = 2x - 4 ', 'x = 3'],
+    ['2x/3 = 2x - 4', 'x = 3'],
     ['(-2/3)x + 3/7 = 1/2', 'x = -3/28'],
-    ['-9/4v + 4/5 = 7/8 ', 'v = -1/30'],
+    ['-(9/4)v + 4/5 = 7/8 ', 'v = -1/30'],
```

Comment:
because of the negative fraction issue mentioned in https://github.com/josdejong/mathjs/issues/1431, I change the initial wrapping of the fraction